lib/nebulex/adapters/local.ex

```diff
@@ -864,4 +897,31 @@ defmodule Nebulex.Adapters.Local do
   defp update_stats(value, _action, _counter_ref) do
     value
   end
+
+  if Code.ensure_loaded?(:telemetry) do
+    defp log_telemetry_start(action) do
+      start_time = System.os_time(:nanosecond)
+
+      :telemetry.execute(
+        [:nebulex, :start],
```

Comment:
At this point, by default, the telemetry prefix should be `[:nebulex, :cache]`, so the events would be `[:nebulex, :cache, :start]`, and in the metadata I would like to have:

- The cache name.
- The action as you have already.
- Maybe the adapter?

All those values you can get them from the adapter metadata.